rust/src/importer.rs

```diff
@@ -144,6 +163,24 @@ impl RpmImporter {
 
         opt_entries.chain(varlib_entries).collect()
     }
+
+    /// Return whether 'doc' files get filtered during importing.
+    pub fn doc_files_are_filtered(&self) -> bool {
+        self.doc_files.is_some()
+    }
+
+    pub fn doc_files_insert(self: &mut RpmImporter, path: &str) {
+        if let Some(ref mut set) = self.doc_files {
```

Comment:
Personally I prefer `self.doc_files.as_mut()` instead of `ref mut` since it's something one can use outside of match patterns too.

Reply:
Ah, the reasoning makes sense, just that I I've rarely written it in that way so far. I'll try to keep that in mind.